server/base/pom.xml

```
@@ -80,6 +80,10 @@
       <groupId>org.apache.hadoop</groupId>
       <artifactId>hadoop-client-api</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.apache.hadoop</groupId>
+      <artifactId>hadoop-azure</artifactId>
```

This could be marked optional:

```suggestion
      <artifactId>hadoop-azure</artifactId>
      <optional>true</optional>
```